sdk/search/Azure.Search.Documents/src/autorest.md

```diff
@@ -95,3 +95,12 @@ modelerfour:
     override:
       "@odata.type": ODataType
 ```
+
+### Disable parameter grouping
+
+AutoRest C# supports parameter grouping now, temporary disabling to reduce the change size. 
+
+``` yaml
+modelerfour:
+  group-parameters: false
```

Comment:
cc @heaths @tg-msft I don't know if you want this feature enabled, try it after I merge the PR and see if the generated code is resonable.

Reply:
We probably don't - thanks for the heads up.

Reply:
FYI it's defined in swagger https://github.com/Azure/azure-rest-api-specs/blob/master/specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2019-05-06-preview/searchindex.json#L99

Reply:
Yep - and I think we'll like how we structured things by hand, but can try it out to see.